src/components/TileCatalog/_catalog-content.scss

```
@@ -22,7 +22,7 @@
 }
 
 .#{$iot-prefix}--sample-tile-title {
-  color: #3d70b2;
+  color: $inverse-support-04;
```

This color is actually a holdover from the old Carbon v9 style. We should update it to be in line with Carbon v10. I think the link text color applied in the ghost buttons makes the most sense.

```suggestion
  color: $link-01;
```